cli/internal/git.go

```diff
@@ -141,14 +143,21 @@ func Clone(repositories ...Project) {
 func GetBranch(gitRepositoryDir string) string {
 	args := []string{"rev-parse", "--abbrev-ref", "HEAD"}
 
-	return Execute(gitRepositoryDir, "git", args[0:]...)
+	branch, err := shell.Execute(gitRepositoryDir, "git", args[0:]...)
+	if err != nil {
+		return ""
+	}
+	return branch
 }
 
 // GetRemote returns the remote from a git repository
 func GetRemote(gitRepositoryDir string, gitDomain string) string {
 	args := []string{"remote", "get-url", "origin"}
 
-	remote := Execute(gitRepositoryDir, "git", args[0:]...)
+	remote, err := shell.Execute(gitRepositoryDir, "git", args[0:]...)
+	if err != nil {
+		return ""
```

Comment:
Same comment as about but with L56

Reply:
Same here: I reviewed that part and noticed it's dead code that I've removed in last commit (ba1e5d5f166f2c260bd175268681e07b7aa99cef)